Ignite project creation is broken on Windows

### Describe the bug

```
✅ What bundle identifier? · com.pizzaappnpx
✅ Where do you want to start your project? · C:\Users\cmiki\Documents\Projects\PizzaAppNpx
✅ How do you want to manage Native code? · CNG
✅ Do you want to initialize a git repository? (Y/n) · Yes
✅ Remove demo code? We recommend leaving it in if it's your first time using Ignite (y/N) · No
✅ Which package manager do you want to use? · pnpm
✅ Do you want to install dependencies? (Y/n) · No
✅ [Experimental] Expo Router for navigation? (y/N) · No
✅ [Experimental] the New Architecture? (y/N) · No

<Ignite ASCII art>

    █ Creating PizzaAppNpx using Ignite 10.0.1
    █ Powered by  ∞ Infinite Red  (https://infinite.red)
    █ Package Manager: pnpm
    █ Bundle identifier: com.pizzaappnpx
    █ Path: C:\Users\cmiki\Documents\Projects\PizzaAppNpx
    ────────────────────────────────────────────────────

    🖨   3D-printing a new React Native app
    🎨 Getting those last few details perfect
    ⚙️ Configuring app.json
    🛠️  Removing fancy demo markup

The following error occurred:

    Error: Command failed: rm -rf src
'rm' is not recognized as an internal or external command,
operable program or batch file.


    Consider opening an issue with the following information at:
    https://github.com/infinitered/ignite/issues/new?template=bug_report.yml&labels=bug
```

Node uses CMD to execute `system.run` commands on Windows, but  `rm` and `mv` are not available in CMD.

I've come up with a PR to resolve this issue -- see below.

### Ignite version

10.0.1

### Additional info

```
System
  platform           win32
  arch               x64
  cpu                16 cores     11th Gen Intel(R) Core(TM) i7-11800H @ 2.30GHz
  directory          Projects     C:\Users\cmiki\Documents\Projects

JavaScript
  node               20.11.1                                   C:\Program Files\nodejs\node.EXE
  npm                10.2.4                                    C:\Program Files\nodejs\npm.CMD
  yarn               1.22.21                                   C:\Program Files\nodejs\yarn.CMD
  pnpm               8.15.3                                    C:\Program Files\nodejs\pnpm.CMD
                     <no pnpm global package info available>
  bun                -                                         not installed
  expo               -                                         not installed

Ignite
  ignite-cli         10.0.1       C:\Users\cmiki\AppData\Local\npm-cache\_npx\e31027f3785124a8\node_modules\.bin\ignite.CMD
  ignite src         build        C:\Users\cmiki\AppData\Local\npm-cache\_npx\e31027f3785124a8\node_modules\ignite-cli\build

Android
  java               17.0.12      C:\Program Files\Eclipse Adoptium\jre-17.0.12.7-hotspot\bin\java.EXE
  android home       -            undefined

Tools
  git                git version 2.43.0.windows.1   C:\Program Files\Git\cmd\git.EXE
```

Thanks for this contribution!

The default Ignite `new` path works on Windows:
```
npx ignite-cli new PizzaApp \
        --bundle=com.pizzaapp \
        --git \
        --install-deps=false \
        --packager=npm \
        --target-path=C:\Users\joshu\Code\PizzaApp \
        --remove-demo=false \
        --workflow=CNG \
        --no-timeout=false \
        --state=mst
```

But I'm having some issues with the Expo Router workflow. 

```
npx ignite-cli new ExpoRouterApp \
        --bundle=com.exporouterapp \
        --git \
        --install-deps=false \
        --packager=npm \
        --target-path=C:\Users\joshu\Code\ExpoRouterApp \
        --remove-demo=false \
        --experimental=expo-router \
        --workflow=CNG \
        --no-timeout=false \
        --state=mst
```

When I try to run any expo commands, I get

```
CommandError: Cannot find module 'ajv/dist/compile/codegen'
Require stack:
- C:\Users\joshu\Code\ExpoRouterApp\node_modules\ajv-keywords\dist\definitions\typeof.js
```

and the clean up commands seem not to be working, this file should be removed

![image](https://github.com/user-attachments/assets/faa3b5d2-8689-41dd-9398-91092a62fb72)

--- a/src/commands/new.ts
+++ b/src/commands/new.ts
@@ -842,16 +842,19 @@ module.exports = {
          * 4. Create a screen template that makes sense for Expo Router
          * 5. Clean up - move ErrorBoundary to proper spot and remove unused files
          */
-        await system.run(log(`mv app/* src/`))
+        filesystem
+          .cwd(targetPath)
+          .find("app")
+          .forEach((file) => filesystem.cwd(targetPath).move(file, file.replace("app", "src")))
         updateExpoRouterSrcDir(toolbox)
         refactorExpoRouterReactotronCmds(toolbox)
         createExpoRouterScreenTemplate(toolbox)
-        await cleanupExpoRouterConversion(toolbox)
+        cleanupExpoRouterConversion(toolbox, targetPath)
 
         stopSpinner(expoRouterMsg, "🧭")
       } else {
         // remove src/ dir since not using expo-router
-        await system.run(log(`rm -rf src`))
+        filesystem.cwd(targetPath).remove("src")
       }
       // #endregion
 

--- a/src/tools/react-native.ts
+++ b/src/tools/react-native.ts
@@ -327,27 +327,19 @@ export function updateExpoRouterSrcDir(toolbox: GluegunToolbox) {
   })
 }
 
-export async function cleanupExpoRouterConversion(toolbox: GluegunToolbox) {
-  const { system, parameters, print } = toolbox
+export function cleanupExpoRouterConversion(toolbox: GluegunToolbox, targetPath: string) {
+  const { filesystem } = toolbox
 
-  // debug?
-  const debug = boolFlag(parameters.options.debug)
-  const log = <T = unknown>(m: T): T => {
-    debug && print.info(` ${m}`)
-    return m
-  }
-
-  await system.run(
-    log(`
-      \\rm src/app.tsx
-      mkdir src/components/ErrorBoundary
-      mv src/screens/ErrorScreen/* src/components/ErrorBoundary
-      rm App.tsx
-      rm ignite/templates/screen/NAMEScreen.tsx.ejs
-      rm -rf ignite/templates/navigator
-      rm -rf src/screens
-      rm -rf src/navigators
-      rm -rf app
-    `),
+  const workingDir = filesystem.cwd(targetPath)
+  workingDir.cwd("src").remove("app.tsx")
+  workingDir.move(
+    workingDir.path("src", "screens", "ErrorScreen"),
+    workingDir.path("src", "components", "ErrorBoundary"),
   )
+  workingDir.remove("App.tsx")
+  workingDir.remove(workingDir.path("ignite", "templates", "screen", "NAMEScreen.tsx.ejs"))
+  workingDir.remove(workingDir.path("ignite", "templates", "navigator"))
+  workingDir.remove(workingDir.path("src", "screens"))
+  workingDir.remove(workingDir.path("src", "navigators"))
+  workingDir.remove("app")
 }

